apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -69,23 +105,78 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
     }
   };
 
+  const iconValue: FontAwesomeV6IconProps = {
+    iconName: 'lock',
+    iconStyle: 'solid',
+  };
+
+  const tabs = [
+    {
+      value: 'viewStudentChatHistory',
+      text: `View ${selectedStudentName}'s chat history`,
+      tabContent: (
+        <div>Viewing {selectedStudentName}'s chat history - TODO</div>
```

Comment:
If this is going out to teacher(s) during our current pilot, we should probably have a more official message here and/or just hide these tabs behind an experiment flag until the pilot ends. Might be good to double check with @samantha-code

Reply:
I wasn't planning to merge this PR until we had the actual chat history in place. We could use an experiment flag though. Do you have a preference?

Reply:
Oh yeah - good point. I can add an experiment flag then since there are a handful of tasks that need to be completed by hopefully different folks and are a bit intertwined.